### PR TITLE
Added heroku qa link for 4pm-2

### PR DESCRIPTION
**Pull request by kthanigaivelan**
In this PR, we added the link to the Heroku QA deployment.

### CODE REVIEW

**Review by archananeupane — APPROVED** [review]
looks good!

File: README.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 [![codecov](https://codecov.io/gh/ucsb-cs156-w22/team04-w22-5pm-courses/branch/main/graph/badge.svg?token=LvEIQ6tYti)](https://codecov.io/gh/ucsb-cs156-w22/team04-w22-5pm-courses)
 
+Heroku QA App Link: https://s22-4pm-2-courses-qa.herokuapp.com/
```
Comment (andrewhlu) [inline README.md:5]: Hi! In addition to adding the link to the deployed app on Heroku, can you also add a link to the Heroku dashboard?

[See relevant section of proj01 instructions](https://ucsb-cs156.github.io/s22/lab/proj01/#:~:text=Make%20an%20issue%2C%20branch%2C%20and%20PR%20to%20a%20link%20to%20the%20deployed%20app%20and%20a%20link%20to%20the%20Heroku%20dashboard%20into%20the%20README.md%20for%20the%20repo.)

**Review by andrewhlu — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (AvivSamet) [inline README.md:5]: I would also label the links as 4pm-2 dashboard and 4pm-2 deployment or something along those lines.

Comment (kthanigaivelan) [reply README.md]: I added that!

File: README.md [hunk 2]
```diff
@@ -2,6 +2,9 @@
 
 [![codecov](https://codecov.io/gh/ucsb-cs156-s22/s22-4pm-courses/branch/main/graph/badge.svg?token=8mmcEy6jrV)](https://codecov.io/gh/ucsb-cs156-s22/s22-4pm-courses)
 
+Heroku QA Dashboard: https://dashboard.heroku.com/apps/s22-4pm-2-courses-qa
+Heroku QA Deployment: https://s22-4pm-2-courses-qa.herokuapp.com/
```
Comment (andrewhlu) [inline README.md]: I just tried accessing this and it looks like there's no frontend due to the missing production flag. You might want to add that in!

File: README.md [hunk 3]
```diff
@@ -2,6 +2,9 @@
 
 [![codecov](https://codecov.io/gh/ucsb-cs156-s22/s22-4pm-courses/branch/main/graph/badge.svg?token=8mmcEy6jrV)](https://codecov.io/gh/ucsb-cs156-s22/s22-4pm-courses)
 
+* Heroku QA Dashboard (4pm-2): https://dashboard.heroku.com/apps/s22-4pm-2-courses-qa
+* Heroku QA Deployment (4pm-2): https://s22-4pm-2-courses-qa.herokuapp.com/
```
Comment (andrewhlu) [inline README.md:6]: I just tried accessing this deployment but I don't see a frontend. You might want to set the production flag :)

**Review by andrewhlu — APPROVED** [review]
Looks good! One suggestion below

File: README.md (hunk 3 above)
Comment (pconrad) [reply README.md:6]: Fixed!

**Review by pconrad — APPROVED** [review]
LGTM!

**Comment by Morgan6688** [comment]
LGTM!

**Review by Morgan6688 — COMMENTED** [review]
LGTM!

**Review by Morgan6688 — APPROVED** [review]

**Pull request merged**